TestApp/android/app/src/main/cpp/native-lib.cpp

```diff
@@ -8,6 +8,7 @@
 #include <string.h>
 #include <vector>
 #include "android/log.h"
+#include <cstdlib> 
```

Comment:
why is this needed?

Reply:
I had an error locally, but I tested it again, and it doesn't seem to be happening anymore, so I removed that line